src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -236,5 +186,58 @@ private static void PublishFiles(IEnumerable<LibraryAsset> files, string outputP
                 File.Copy(file.ResolvedPath, Path.Combine(outputPath, Path.GetFileName(file.ResolvedPath)), overwrite: true);
             }
         }
+
+        public static int Main(string[] args)
```

Comment:
I changed my mind. This should be at the top and the publish command should be in another class. Let's do that across the board.